orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/sql/pipeline/persistence/SqlExecutionRepository.kt

```diff
@@ -443,14 +426,17 @@ class SqlExecutionRepository(
     val select = jooq.selectExecutions(
       PIPELINE,
       conditions = {
-        val inClause = "config_id IN (${pipelineConfigIds.joinToString(",") { "'$it'" }})"
-        val timeCondition = it.where(inClause)
+        var conditions = it.where(
+          field("config_id").`in`(*pipelineConfigIds.toTypedArray())
```

Comment:
this cute syntax is needed because kotlin for whatever reason can't disambiguate the calls